stock_quant_partner_info/models/stock_quant.py

```diff
@@ -14,8 +14,12 @@ class StockQuant(models.Model):
 
     @api.depends('location_id', 'history_ids')
```

Comment:
@gurneyalex You can improve your depends with 

```python
'history_ids.picking_id.partner_id.commercial_partner_id', 'history_ids.location_id.usage'
```
Avoiding fetching data inside function